src/app/components/FrostedGlassPromo/index.tsx

```diff
@@ -54,7 +54,7 @@ const FrostedGlassPromo = ({
   const isCanonical = !isAmp;
   const relativeUrl = makeRelativeUrlPath(url);
 
-  const clickTracker = useClickTrackerHandler({
+  const { onClick: clickTracker } = useClickTrackerHandler({
```

Comment:
Does this mean that there won't be any click tracking for Frosted Promos on lite? I wonder if we should refactor this file to allow it?